### ISSUE TITLE
Waveplot not implemented with MPI

### ISSUE THREAD

**Issue by lhargreaves95** [dftbplus/dftbplus#197]
Hello,

I am having trouble plotting the cube files using wave plot. It appears that it has not been parallelised. Currently when submitting the job with N number of cores, it submits the job N number of times.

Thanks

**Comment by bhourahine** [comment]
yes, waveplot is not currently parallelized with MPI. Is this a limiting factor for plotting your data (a serial compilation of waveplot will read the same data files)?

**Comment by lhargreaves95** [comment]
Yes, I am trying to perform Bader analysis for a molecule adsorbed on a periodic slab. Running on one core for a grid that is fine enough, takes a very long time.
